spec/models/sms_ten_dlc_brand_spec.rb

```diff
@@ -25,209 +25,1311 @@
+      expect(instance.entity_type).to eq("PRIVATE_PROFIT")  
+    end
+    it 'assigns value PUBLIC_PROFIT' do
+      instance.entity_type = "PUBLIC_PROFIT"
+      expect(instance.entity_type).to eq("PUBLIC_PROFIT")  
+    end
+    it 'assigns value NON_PROFIT' do
+      instance.entity_type = "NON_PROFIT"
+      expect(instance.entity_type).to eq("NON_PROFIT")  
+    end
+    it 'assigns value GOVERNMENT' do
+      instance.entity_type = "GOVERNMENT"
+      expect(instance.entity_type).to eq("GOVERNMENT")  
+    end
+    it 'assigns value SOLE_PROPRIETOR' do
+      instance.entity_type = "SOLE_PROPRIETOR"
+      expect(instance.entity_type).to eq("SOLE_PROPRIETOR")  
+    end
+    it 'throws error on invalid enum' do
+      instance = Freeclimb::SMSTenDLCBrand.new
+      expect{instance.entity_type = "INVALID_ENUM"}.to raise_error(ArgumentError)
     end
   end
 
+
   describe 'test attribute "csp_id"' do
     it 'should work' do
-      # assertion here. ref: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers
+      instance.csp_id = "TEST_STRING"
+      expect(instance.csp_id).to eq("TEST_STRING")  
     end
   end
 
+
   describe 'test attribute "brand_id"' do
     it 'should work' do
-      # assertion here. ref: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers
+      instance.brand_id = "TEST_STRING"
+      expect(instance.brand_id).to eq("TEST_STRING")  
     end
   end
 
+
   describe 'test attribute "first_name"' do
     it 'should work' do
-      # assertion here. ref: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers
+      instance.first_name = "TEST_STRING"
+      expect(instance.first_name).to eq("TEST_STRING")  
     end
   end
 
+
   describe 'test attribute "last_name"' do
     it 'should work' do
-      # assertion here. ref: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers
+      instance.last_name = "TEST_STRING"
+      expect(instance.last_name).to eq("TEST_STRING")  
     end
   end
 
+
   describe 'test attribute "display_name"' do
     it 'should work' do
-      # assertion here. ref: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers
+      instance.display_name = "TEST_STRING"
+      expect(instance.display_name).to eq("TEST_STRING")  
     end
   end
 
+
   describe 'test attribute "company_name"' do
     it 'should work' do
-      # assertion here. ref: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers
+      instance.company_name = "TEST_STRING"
+      expect(instance.company_name).to eq("TEST_STRING")  
     end
   end
 
+
   describe 'test attribute "ein"' do
     it 'should work' do
-      # assertion here. ref: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers
+      instance.ein = "TEST_STRING"
+      expect(instance.ein).to eq("TEST_STRING")  
     end
   end
 
+
   describe 'test attribute "ein_issuing_country"' do
```

Comment:
This appears more than once in the test file, and I think it repeats for a couple of other items in the file